ASM/c/dungeon_info.c

```diff
@@ -230,8 +234,10 @@ void draw_dungeon_info(z64_disp_buf_t *db) {
                 if (total_keys > 9) total_keys = 9;
 
                 char count[5] = "O(O)";
-                if (current_keys > 0) count[0] = current_keys + '0';
-                if (total_keys > 0) count[2] = total_keys + '0';
+                if (SHUFFLE_CHEST_GAME) {
```

Comment:
It would probably be better to only show the chest game row if this setting is on. In any case, this conditional doesn't make sense as it's being applied to all rows.

Reply:
That is actually the way I had it before but because of the way the symbol is written to the rom, it will still show when keysy is on. But I see what you mean otherwise and ill fix it

Reply:
You could set the symbol to `2` in keysy and compare it to `1` here.

Reply:
oh so in patches.py, I would refactor the code to say:

```
if world.settings.shuffle_tcgkeys != 'vanilla':
        if 'keysy' in world.settings.shuffle_tcgkeys:
                rom.write_byte(rom.sym('SHUFFLE_CHEST_GAME'), 0x02)
        else:
                rom.write_byte(rom.sym('SHUFFLE_CHEST_GAME'), 0x01)
        # Update Chest Game Salesman to better fit the fact he sells a randomized item
        update_message_by_id(messages, 0x6D, "I seem to have misplaced my\x01keys, but I have a fun item to\x01sell instead.\x04How about \x05\x4110 Rupees\x05\x40?\x01\x01\x1B\x05\x42Buy\x01Don't Buy\x05\x40\x02")
        update_message_by_id(messages, 0x2D, "That's OK!\x01More fun for me.\x02")
        update_message_by_id(messages, 0x6E, "Wait, that room was off limits!\x02")
        update_message_by_id(messages, 0x704C, "I hope you like it!\x02")
```

Sorry for the questions, I'm still learning lol